apps/remote_control/lib/lexical/remote_control/search/store/backends/ets/schema.ex

```diff
@@ -54,8 +76,15 @@ defmodule Lexical.RemoteControl.Search.Store.Backends.Ets.Schema do
     end
   end
 
-  alias Lexical.Project
-  alias Lexical.VM.Versions
+  @spec entries_to_rows(Enumerable.t(Entry.t()), module()) :: [tuple()]
+  def entries_to_rows(entries, schema_module) do
```

Comment:
This PR should also be helpful for my Rename PR, so `entries_to_rows` is a general operation, right?

Reply:
Yes.